### ISSUE TITLE
ListView. Warning: Failed prop type: ListView: prop type `viewConfiguration.columnConfiguration[0].title` is invalid; 

### ISSUE THREAD

**Issue by andresavellan** [lumada-design/hv-uikit-react#2695]
### Latest version

- [X] I have tested the latest version

### No Hitachi Vantara confidential content

- [X] I'm not including any Hitachi Vantara confidential content

### Current behavior 😯

Nothing visually or functional but a warning in the console when using AssetInventory and ListView. Console: Warning: Failed prop type: ListView: prop type `viewConfiguration.columnConfiguration[0].title` is invalid; 

### Expected behavior 🤔

Nothing in the console. 

### Steps to reproduce 🕹

Steps:

1. https://lumada-design.github.io/uikit/master/?path=/docs/components-asset-inventory--main
2. Or, any ListView in documentation. 
3.
4.

### Context 🔦

Just want to get rid of the warning. 

### Your Environment 🌎

Device: MacBook pro
OS: Big Sur 11.6
Browser Chrome 99.0.4844.51 (Official Build) (x86_64), FireFox 98.0.1 (64-bit), Edge Version 99.0.1150.46 (Official build) (x86_64)

**Comment by francisco-guilherme** [comment]
Hi @andresavellan, tks for reaching out.

In want king of environment did you identified that issue. Were you in dev mode?
I'm not seeing it on the link you provided: [asset-inventory](https://lumada-design.github.io/uikit/master/?path=/docs/components-asset-inventory--main)

Anyway, we are currently working on a more flexible and valuable way of implementing the asset inventory pattern. 
Stay tuned 👍

**Comment by andresavellan** [comment]
Hello @francisco-guilherme, and thank you.

The link I provided is the code (Asset Inventory/Main) I copied to create cards and list. 
I have also tried Asset Inventroy/List View [()](https://lumada-design.github.io/uikit/master/?path=/docs/components-asset-inventory-list-view--main)

Yes I was on dev mode and I see now that it works fine in production. 👍

Looking forward to the updates.

Thank you again, keep up the good work.
